input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+AU Core and IPA share a compatible approach to handling missing data through the use of the DataAbsentReason extension.
+
+##### Suppressed Data
+AU Core provides additional guidance for handling data suppression due to security or privacy reasons, using 'unknown' or 'masked' codes from the DataAbsentReason Code System based on the requester's access rights.
+This ensures AU Core resources with suppressed data remain compliant with IPA through its use of the DataAbsentReason extension for missing data.
+
+##### Additional profiles
+This version of AU Core has no equivalent profile for the following IPA profiles:
+- IPA-DocumentReference
+- IPA-MedicationStatement
+
+#### International Patient Summary
+[International Patient Summary Implementation Guide 1.1.0](https://hl7.org/fhir/uv/ips/STU1.1/) describes how to represent the International Patient Summary (IPS) using HL7 FHIR. An International Patient Summary  document is an electronic health record extract containing essential healthcare information about a subject of care.
+
+AU Core is designed to be compatible with IPS, meaning AU Core data can be used to create a patient summary that aligns with IPS standards. To achieve compliance with both AU Core and IPS, additional requirements may need to be met. These requirements are outlined below.
+
+<table border="1" style="width: 100%; margin: auto; border-collapse: collapse;">
+    <thead>
+        <tr>
+            <th style="width: 25%;">AU Core</th>
+            <th style="width: 25%;">International Patient Summary 1.1.0</th>
+            <th style="width: 25%;">Element</th>
+            <th style="width: 25%;">Additional requirements</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-AllergyIntolerance-uv-ips.html">Allergy Intolerance (IPS)</a></td>
+            <td style="width: 25%;">AllergyIntolerance.patient.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Condition-uv-ips.html">Condition (IPS)</a></td>
+            <td style="width: 25%;">Condition.clinicalStatus</td>
+            <td style="width: 25%;">IPS requires minimum = 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Condition.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-diagnosticresult.html">AU Core Diagnostic Result Observation</a></td>
```

Comment:
Missing entries for Observation.value for Diagnostic & Path. Should say "IPS requires UCUM for coded quantity units."

Reply:
Done

Reply:
Applied to Diagnostic but Missing for Path.

Reply:
I've checked this a couple of times as I think it's possibly a typo creating confusion....maybe...But I think the requirements is missing for the US Core equivalent of Path.
1. AU Core Diagnostic Result Observation --> Observation Result (IPS)
   - this additional requirement does not apply
 2. AU Core Pathology Result Observation --> Observation Results: laboratory (IPS) and -->  Observation Results: pathology (IPS)
   - this additional requirement does apply and is included for each compared profile
3. AU Core Diagnostic Result Observation --> US Core Observation Clinical Result
  -   this additional requirement applies and is included 
4. AU Core Pathology Result Observation --> US Core Laboratory Result Observation Profile
   - replaced green with a yellow tick in the comparison table
   - added a row for AU Core Pathology Result Observation in the US Core Additional requirements table

Reply:
Yes it was the US Core Pathology not IPS. Noted the changes.